src/lib.rs

```diff
@@ -131,22 +130,31 @@ impl Anchor {
     }
 }
 
-/*
-philtr -> tweets
+fn agent_anchor() -> ZomeApiResult<Address> {
```

Comment:
This is just an anchor though? Why not just add it to you're app. Not everyone who uses anchors would need a special "Agents" anchor right?